evalml/model_understanding/__init__.py

```diff
@@ -1,20 +1,24 @@
 from .graphs import (
-    precision_recall_curve,
-    graph_precision_recall_curve,
-    roc_curve,
-    graph_roc_curve,
-    graph_confusion_matrix,
+    binary_objective_vs_threshold,
```

Comment:
Reordered imports alphabetically

Reply:
Did flake8 recommend this?

Reply:
@freddyaboulton Yeah it weirdly did and it came out of nowhere! One day it wasn't recommending it and the next it was. Correct me if I'm wrong but it makes sense right, as far as alphabetization is concerned?